<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SwitchModeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence diagrams</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="buttons">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>121</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="AddTimelineButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add timeline</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SyncMessageButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Sync message</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AsyncMessageButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>290</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Async message</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ReturnMessageButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Return message</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="CreateMessageButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Create message</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteMessageButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete message</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="SqDeleteItemButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>380</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete items</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="NewDiagramButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>New diagram</string>
      </property>
     </widget>
     <widget class="QComboBox" name="DiagramList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteDiagramButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete diagram</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="CdDeleteItemButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete items</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AddClassButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add class</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddGenerButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>generalizace</string>
      </property>
      <property name="text">
       <string>Generalization</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>50</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AddAsocButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string notr="true">asociace</string>
      </property>
      <property name="text">
       <string notr="true">Asociation</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>50</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AddAggrButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>agregace</string>
      </property>
      <property name="text">
       <string>Aggregation</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>50</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AddCompButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>kompozice</string>
      </property>
      <property name="text">
       <string>Composition</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>50</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="views">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>9</y>
      <width>1771</width>
      <height>981</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_5">
     <widget class="QGraphicsView" name="classDiagramView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1760</width>
        <height>980</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QGraphicsView" name="seqDiagramView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1760</width>
        <height>980</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1760</width>
        <height>980</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1760</width>
        <height>980</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
